framework/proto/flwr/proto/control.proto

```diff
@@ -86,3 +96,15 @@ message StopRunResponse { bool success = 1; }
 
 message PullArtifactsRequest { uint64 run_id = 1; }
 message PullArtifactsResponse { optional string url = 1; }
+
+message AddNodeRequest { string pub_key = 1; }
+message AddNodeResponse { optional uint64 node_id = 1; }
+
+message RemoveNodeRequest { uint64 node_id = 1; }
+message RemoveNodeResponse {}
+
+message ListNodesRequest {}
+message ListNodesResponse {
+  map<uint64, NodeInfo> node_dict = 1;
+  string now = 2;
```

Comment:
Is `now` the SuperLink timestamp? What would we use it for?

Reply:
yes, it's the timestamp when the response is created. in `flwr ls` we use it to compute the elapsed time for an ongoing run before displaying the table (https://github.com/adap/flower/blob/757c9fae2b56522624733a59efded61f3e63594d/framework/py/flwr/cli/ls.py#L194). I was thinking keeping this would be useful to potentially show how long a `SuperNode` has been connected for.